dask/highlevelgraph.py

```diff
@@ -931,6 +931,8 @@ def cull(self, keys: Iterable) -> HighLevelGraph:
         hlg: HighLevelGraph
             Culled high level graph
         """
+        from dask.layers import Blockwise
```

Comment:
I'd rather not import Blockwise like this, but it also feels wasteful not to use culled_deps for the most common case where the layer and key dependencies will be the same. Open to suggestions here.